pootle/apps/pootle_project/models.py

```diff
@@ -301,6 +301,11 @@ def config(self):
         return ObjectConfig(self)
 
     @property
+    def local_fs_path(self):
+        return os.path.join(
+            settings.POOTLE_FS_PATH, self.code)
```

Comment:
no atm its only in test fixtures, but tbh this class is not yet used, so i would rather add the default/docs in pootle_fs_merge